packages/pages/src/components/analytics/hooks.ts

```diff
@@ -27,8 +27,8 @@ export function useAnalytics(): AnalyticsMethods {
 
   // TODO: is this the right way / place to expose a callback for use by a Cookie Management banner?
```

Comment:
Do you want to remove this TODO now?

Reply:
I mean I am still unclear if this is the best way. Product also mentioned potentially checking for a cookie as a future thing they may want to do. 

What is your philosophy on TODOs? I do not consider them merge blockers generally.  I can also create a corresponding issue in Github to keep track, rather than just leaving them in the code, if you'd prefer.

Reply:
Spoke offline, going to resolve this conversation and leave the TODO in place.